openedx/core/djangoapps/performance/tests/test_logs.py

```diff
@@ -1,4 +1,7 @@
-"""Tests that performance data is successfully logged."""
+# pylint: disable=no-member
```

Comment:
Can you put this just on the lines that need it? I don't like disabling PyLint checks for a whole file.